### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by ohkhaila**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -3,39 +3,85 @@
...
+              <a id="about" class="smooth"></a>
+              <h1> Keeping Up with Kay </h1>
+              <h2> Hello! Welcome!  </h2>
+              <h3> Intro Video</h3>
+                <!--About page content goes here.-->
+            </div>
+
+            <div id="page2">
+              <a id="blog" class="smooth"></a>
+              <h4> Latest blog posts<br/>
```
Comment (tilikang) [inline index.php:46]: This is nitpicky, but I probably wouldn't put all the posts inside an h4 tag. h4 implies it's going be some kind of header text rather then a big section of links. If you want it to be styled like an h4, wrap it in a div, and apply some CSS to that div.

File: style.css [hunk 2]
```diff
@@ -17,28 +94,28 @@ body{
             color:#000000;
             background-color: #b1dfd5;
             width: 100%;
-            height:200px;
+            height:500px;
```
Comment (tilikang) [inline style.css:97]: Having these heights on the h1-h5 tags seems a bit unusual. What are you trying to accomplish with this?

**Review by tilikang — COMMENTED** [review]
Looks good. Am I correct that behavior.php is the main thing you're working on right now? If so it seems like the next step is to figure out what to do when the form is submitted. Let me know if you need help with anything

File: style.css (hunk 2 above)
Comment (ohkhaila) [reply style.css:97]: I didn't know that "h" was for header. "h1-h5" is the blocks of content on my site. I wanted to group different aspects together. Now I'm thinking that "div" would be better for this.

Comment (tilikang) [reply style.css:97]: Yeah, it sounds like a div would be best. The other tags have specific meanings (h1 = header 1, p = paragraph, etc.) whereas a div is just a generic container that can be styled however you want.

**Pull request merged**